### ISSUE TITLE
Stacktrace at wall creation 

### ISSUE THREAD

**Issue by EstebanDugueperoux2** [IfcOpenShell/IfcOpenShell#5884]
Hi ifcopenshell community,

I try to follow https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.8.0/src/bonsai/docs/tutorials/project0.rst tutorial, which doesn't seems visible on official website (https://docs.bonsaibim.org/).

I use latest unstable installation (https://raw.githubusercontent.com/IfcOpenShell/bonsai_unstable_repo/main/index.json), bonsai_py311-0.8.1-alpha241212-linux-x64.zip at this date, with blender-4.3.1-linux-x64.

I start a new Metric Project, I select Z axis on gizmo, then:
- select "Wall Tool", then "Quick Create IfcWallType";
- click on "Quick Create IfcWallType";
- set RL to 0.2m for the tickness of the wall If I understand correctly;
- set H to 2.8m;
- click on Add;
- click a first time inside the 3D Viewpoint;
- Alt+Tab to switch temporarily to another window and come back
- click a second time inside the 3D Viewpoint and a popup appears with following stacktrace: 
![Capture d’écran du 2024-12-17 21-19-01](https://github.com/user-attachments/assets/bc0c262d-11f4-46dd-b97c-beaeb6468c7d)

```
Traceback (most recent call last):
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/bim/module/model/wall.py", line 389, in modal
    self.handle_inserting_polyline(context, event)
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/bim/module/model/polyline.py", line 274, in handle_inserting_polyline
    result = tool.Polyline.insert_polyline_point(self.input_ui, self.tool_state)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 464, in insert_polyline_point
    d = input_ui.get_formatted_value("D")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 68, in get_formatted_value
    value = self.get_number_value(attribute_name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 63, in get_number_value
    return float(value)
           ^^^^^^^^^^^^
ValueError: could not convert string to float: '7.000 m'
Error: Python: Traceback (most recent call last):
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/bim/module/model/wall.py", line 389, in modal
    self.handle_inserting_polyline(context, event)
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/bim/module/model/polyline.py", line 274, in handle_inserting_polyline
    result = tool.Polyline.insert_polyline_point(self.input_ui, self.tool_state)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 464, in insert_polyline_point
    d = input_ui.get_formatted_value("D")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 68, in get_formatted_value
    value = self.get_number_value(attribute_name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/esteban/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/bonsai/tool/polyline.py", line 63, in get_number_value
    return float(value)
           ^^^^^^^^^^^^
ValueError: could not convert string to float: '7.000 m'
```

Regards.

**Comment by steverugi** [comment]
@EstebanDugueperoux2 I usually create an element from the `Quick Create IfcWallType` and adjust its parameters later
in case of a wall I draw its length then go to Object Materials and edit its thickness, when OK I change the height using the top menu `H`

**Comment by brunoperdigao** [comment]
@EstebanDugueperoux2 This issue was fixed by this commit: 8c6eddfb66bb4d7d1bbffb521e4f5127709520ea
When you press `Alt+Tab`, you may inadvertently enter the input UI, which is highlighted in blue. While this UI is active, you won't be able to use the left mouse button. To regain functionality, you need to press `Esc` and then click again.

**Comment by EstebanDugueperoux2** [comment]
Hi @brunoperdigao  ok thanks, I'll test once available in https://github.com/IfcOpenShell/bonsai_unstable_repo
